.gitignore

```diff
@@ -3,6 +3,7 @@ gitignore/
 node_modules/
 public/
 .vscode/
+package-lock.json/
```

Comment:
why is there a package-lock.json file anyways, does it have to commit to this project or can it only exist in your, i see you are the one who added it to the repo,

Reply:
Forgot about that my Intelij is set up to .add my settings every time I checkout,